src/Controller/HomeController.php

```diff
@@ -3,17 +3,52 @@
 namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class HomeController extends AbstractController
 {
     /**
-     * @Route("/", name="home")
+     * @Route("/", name="student_list")
+     * @param KernelInterface $kernel
+     * @return Response
      */
-    public function index()
+    public function index(KernelInterface $kernel)
     {
+        $json = (string)file_get_contents($kernel->getProjectDir().'/public/students.json');
+        $data = json_decode($json, true);
+
+        $projects = $students = [];
+        foreach ($data as $projectDomain => $items) {
+            $projects[] = ['domain' => $projectDomain, 'name' => $items['name']];
+            foreach ($items['students'] as $student) {
+                $students[] = ['name' => $student, 'mentor' => $items['mentors'][0], 'project' => $projectDomain];
```

Comment:
Šiuo metu duomenys yra žinomi iš anksto, bet gera praktika yra nepriklausyti stipriai nuo pradinių duomenų (jei `mentors` yra masyvas, tai tikėtina, kad ateityje ten gali būti ir daugiau elementų).

T.y. geriau `Twig`e padaryti papildomą `for` eilute daugiau, o ne priklausyti nuo konkretaus kiekio masyve. Įpratus taip mąstyti – bus mažiau darbo tikrinti kodą pasikeitus duomenims.